### DISCUSSION TITLE
Tactical RMM inaccessible

### DISCUSSION THREAD

**Discussion by GuilhermePesoa** [amidaware/tacticalrmm#1575] [Q&A]
Hello! I hope they're all right.

After the internet operator changed the IP address made available to the network of my Tactical RMM server, the service went offline even after adjusting the DNS zone with the new address.

I tested the rmm, api and mesh addresses with PING and confirmed that provisioning has already occurred, but my tactical is still inaccessible.

Could you support me with this?
Thanks!

**Comment by dinger1986** [comment]
use dig to check the a records have propegated, check internal dns that its not maybe caching the only details, otherwise are you using a standard install?

**Reply by GuilhermePesoa** [reply]
Thank you for your answer, but unfortunately my knowledge was not enough to understand how I should follow your guidelines.
Can you describe a little more?

**Reply by GuilhermePesoa** [reply]
By DIG I was able to prove that the provisioning has already occurred.
(My Tactical has been offline for three days)

**Reply by dinger1986** [reply]
can you ping and get a response from the machines which are offline? Commercial support is available if that helps?

**Reply by GuilhermePesoa** [reply]
The server responsible for Tactical is online on my local network, but I can't access it through the web address rmm.domain.com

**Reply by dinger1986** [reply]
can you share a link please?

**Reply by GuilhermePesoa** [reply]
This is the address of my tactical rmm: https://rmm.vyrtus.com.br/

**Reply by dinger1986** [reply]
Doesnt work for me, did they replace your router? is port forwarding setup? That domain resolves to 177.140.51.161, is that right?

**Reply by GuilhermePesoa** [reply]
The IP address is correct On my router I have a port forwarding to the server the way we always use it.

**Reply by dinger1986** [reply]
something has changed somewhere

**Reply by GuilhermePesoa** [reply]
The event that culminated in this situation was the change of the public IP provided to my network.
But I tried to handle this change by adjusting my DNS zone

Is there anything I can collect from my server to bring you more information?

**Reply by dinger1986** [reply]
are you just using a standard install? Tactical works off of dns not IPs other than the local IP assigned via the network card. Is your router maybe setup to forward from an external IP to internal? I know thats how mikrotik does it

**Reply by GuilhermePesoa** [reply]
In my Mikrotik router I have a dstnat with TCP protocol using port 443

**Reply by GuilhermePesoa** [reply]
Yes, I'm using the standard installation

**Reply by dinger1986** [reply]
can you share the rule?

**Reply by GuilhermePesoa** [reply]
![qweqasdas](https://github.com/amidaware/tacticalrmm/assets/39242316/11d21005-1bc0-458b-b1b6-1cf0cdc896a2)

**Reply by dinger1986** [reply]
what if you remove the Dst Address? and change it to the dst (wan) port?

**Reply by GuilhermePesoa** [reply]
Previously I had tried this, but I will make this adjustment and perform a new test

**Reply by GuilhermePesoa** [reply]
I made the adjustment, but I still get TIME OUT when accessing the web address

**Reply by dinger1986** [reply]
what happens if you add the local ip in your local computers hosts file and try and connect that way? Has the ip changed of the linux machine?

**Reply by GuilhermePesoa** [reply]
Hello, sorry for my unavailability to respond faster.

When accessing the private address '192.168.0.15' I have the Tactical login screen but with invalid SSL certificate and it is not possible to login "Backend is offline (network error)"

**Reply by dinger1986** [reply]
Did you add it to your hosts?

**Reply by GuilhermePesoa** [reply]
Are you referring to the etc/hosts file?

**Reply by dinger1986** [reply]
I mean the hosts file on whatever of you are using, so using the urls to the rmm server internal ip

**Reply by GuilhermePesoa** [reply]
Sorry, I still don't understand what you mean by "Did you add it to your hosts?"

Please do not give up on me

**Reply by dinger1986** [reply]
The hosts file on your pc you are trying to access tactical from

**Reply by GuilhermePesoa** [reply]
Understood
I added the local address of the Tactical server in my PC's hosts file, but I still get the same return I described a little above

**Reply by dinger1986** [reply]
But did you go to the dns name or IP? Cause it needs to be the dns name

**Reply by GuilhermePesoa** [reply]
In my  host file i added '192.168.0.15', 'rmm.vyrtus.com.br', 'mesh.vyrtus.com.br' and 'api.vyrtus.com.br'

**Reply by dinger1986** [reply]
And then tried to browse to the url?

**Reply by GuilhermePesoa** [reply]
Yes, i tried
By the '192.168.0.15' i have the Tactical login screen but with invalid SSL certificate and it is not possible to login "Backend is offline (network error)"

By the rmm.vyrtus.com.br i have timed out

**Reply by dinger1986** [reply]
Seems something weird is going on. I don't know what else to suggest without looking

**Reply by GuilhermePesoa** [reply]
I understand friend, is there any information that I can provide you to help with the analysis?

**Reply by dinger1986** [reply]
Not really, I can only really suggest commercial support at this point https://docs.tacticalrmm.com/support/#commercial-support can have a look and give you a price to sort it out

**Comment by th3recker** [comment]
i have the same issue, i have followed the docs run the tests my SSL comes back fine everything is online and running, cant login off local network nor online, i see that other users are having the same issue after updating, did we get a bad installer or the devs messed up i dunno, does any1 have a solution for this as of yet ?

**Reply by silversword411** [reply]
Troubleshooting is better done thru Discord because we have a whole list of template questions in #rules to try and figure out why your install isn't working.
Where are you hosting?
What's the DNS name?
etc.

**Comment by th3recker** [comment]
this is 1 of many of the url i have used for testing, to no avail, https://docs.tacticalrmm.com/troubleshooting/
everything is pingable and resolves, this was a clean install on debian done via netinstall package then followed the installation doc to the T and result was backend offline ( network error )

**Comment by th3recker** [comment]
Hi,

Thanks for the peompt reply, i am new to this and have been struggling for
a week to try get it working with multiple re installations...

Could u pls be so kind as to share the discord channel link and ill gladly
join and suport the chennel.

Kindest regards
Shaun

**Reply by wh1te909** [reply]
https://discord.gg/upGTkWp
